hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -146,6 +148,9 @@ const int UBLOX_WAIT_AT_RESPONSE_WHILE_UUFWINSTALL_PERIOD = 10000;
 const unsigned CHECK_SIM_CARD_INTERVAL = 1000;
 const unsigned CHECK_SIM_CARD_ATTEMPTS = 10;
 
+const uint32_t SYSTEM_CACHE_OPERATION_MODE_VALID = 0x97c10000;
```

Comment:
I'm assuming this is just a random magic pattern to ensure the cached values are valid when read back? 
Are we worried about data in the system cache being unreliable?

Reply:
This is mainly to be sure that a value has been set by CEMODE code only and not some other way, "PTCL" :D  I guess the fact that the KEY is found is a similar thing, but it makes me feel better as a double check, and it allowed me to invalidate the data easily while testing.